stream_alert_cli/runner.py

```diff
@@ -41,8 +43,7 @@ def cli_runner(options):
             Contains the following keys for lambda commands:
                 (command, subcommand, env, func, source)
     """
-    cli_load_message = ('Issues? Report here: '
-                        'https://github.com/airbnb/streamalert/issues')
+    cli_load_message = ('Issues? Report here: ' 'https://github.com/airbnb/streamalert/issues')
```

Comment:
I think yapf did this, but the parenths are no longer required, along with the quotes in the middle

Reply:
Yep, it was `yapf`, not me I promise